viz-lib/src/visualizations/chart/plotly/applyLayoutFixes.js

```diff
@@ -76,18 +76,8 @@ function placeLegendBelowPlot(plotlyElement, layout, updatePlot) {
     if (legend) {
       // compute real height of legend - items may be split into few columnns,
       // also scrollbar may be shown
-      const bounds = reduce(
-        legend.querySelectorAll(".traces"),
-        (result, node) => {
-          const b = node.getBoundingClientRect();
-          result = result || b;
-          return {
-            top: Math.min(result.top, b.top),
-            bottom: Math.max(result.bottom, b.bottom),
-          };
-        },
-        null
-      );
```

Comment:
@kravets-levko what's causing the bug is that there is some "margin" for the legends background, computing each of the items separately don't take it into account, so it separating less than needed and cutting part of the text. I made some tests and it seems the `.legend` holds the same information we had before + it also considers the margin for the background size.

I'm continuing to test this change, but so far looks fine, lmk if you see any situation where this may cause a problem.